src/pages/nominate.js

```diff
@@ -52,16 +52,13 @@ class App extends React.Component {
 
   handleSubmit = e => {
     const validation = this.validateName();
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to send your nomination')) {
+      const urlEncodedTwitterHandle = encode(this.state.name);
+      const urlEncodedReason = encode(this.state.reason);
```

Comment:
I don't think it's a blocker, but I found that the form doesn't work for me if I include " 😁" at the end of the text reason—it ends up with a GET against the nominate page including the form args rather than opening a new window.

I imagine this would end up being a pain to hunt down, thus not a blocker, but figured it was worth mentioning in case we ever need to support other languages

Reply:
Ahh yeah. A quick solution I've been able to find for this has been to replace `encode(this.state.reason)` with `encodeURIComponent(this.state.reason)`. It seems that if there's an emoji in the text, `encode` simply returns an empty string, whereas `encodeURIComponent` returns the string and something like ` 0=@` to represent the emoji. The good news is that encoding the string `encodeURIComponent` allows us to actually redirect to the twitter DM page unlike `encode`